charts/openebs/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 1.10.2
+version: 1.11.0
```

Comment:
Can you update the chart version as `1.10.3`. It will be moved to `1.11.0` for the next openebs release.

Reply:
No worries, committed as requested :)